docs/guides/manage-identities/30_account-recovery.mdx

```diff
@@ -3,12 +3,12 @@ id: account-recovery
 title: Administrative account recovery
 ---
 
-It is possible to initiate account recovery for identities using the Admin API endpoints.
+You can initiate account recovery for Identities using the Admin API endpoints.
```

Comment:
there is a few typos in this doc but we can fix them after the PR,

and a link to https://console.ory.sh/projects/current/session that leads nowhere, needs to be updated to https://console.ory.sh/projects/current/session-settings.

also the selfhosted kratos example doesnt make sense. it would be enough to use the cURL example with selfhosted no?

Reply:
I looked for typos and fixes some wonky sentences. For self-hosted it's okay to either edit the config or call the API, idk which one is better.